Sources/SmileID/Classes/SmileID.swift

```diff
@@ -57,6 +59,12 @@ public class SmileID {
         self.useSandbox = useSandbox
         self.apiKey = apiKey
         SmileIDResourcesHelper.registerFonts()
+        Task {
+            // we use the deviceId and not fingerprint as this is unique and does not change
```

Comment:
Will device ID return the same value in different apps? This is not the case on Android so I decided to use fingerprint there

Reply:
Yes it does. The iOS docs say deviceId is unique but fingerprint changes even with os updates